static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -230,35 +231,51 @@ const canShow = async () => {
     }
 };
 
-const show = () => import(
-    /* webpackChunkName: "guardian-braze-components" */ '@guardian/braze-components'
-    )
-    .then((module) => {
-        const container = document.createElement('div');
+const show = () => Promise.all([
+    import('react-dom'),
+    import('@emotion/core'),
+    import('@emotion/cache'),
+    import(/* webpackChunkName: "guardian-braze-components" */ '@guardian/braze-components')
+]).then((props) => {
+    const [{ render }, { CacheProvider }, createCacheModule, brazeModule] = props
+    const container = document.createElement('div');
         container.classList.add('site-message--banner');
 
         // The condition here is to keep flow happy
         if (document.body) {
             document.body.appendChild(container);
         }
 
-        mountDynamic(
-            container,
-            module.BrazeMessage,
-            {
-                componentName: messageConfig.extras.componentName,
-                logButtonClickWithBraze: (buttonId) => {
-                    if (appboy) {
-                        const thisButton = new appboy.InAppMessageButton(`Button ${buttonId}`,null,null,null,null,null,buttonId)
-                        appboy.logInAppMessageButtonClick(
-                            thisButton, messageConfig
-                        );
-                    }
-                },
-                submitComponentEvent,
-                brazeMessageProps: messageConfig.extras,
-            },
-            true,
+        const Component = brazeModule.BrazeMessage
+        const shadowRoot = container.attachShadow({ mode: 'open' });
```

Comment:
I _think_ the usage of shadow DOM needs to be conditional here as it is in `mountDynamic`. Shadow DOM [isn't supported](https://caniuse.com/?search=shadow%20dom) in IE11. When shadow DOM isn't available things are much simpler as we also don't need to use a custom `CacheProvider`.

Reply:
Good point!! I have just pushed an update. I have tested it locally to check and it now works on IE